draft-ietf-masque-connect-ip.md

```diff
@@ -25,7 +25,6 @@ keyword:
   - proxy
   - tunnels
   - quic in udp in IP in quic
-  - turtles all the way down
```

Comment:
Sorry... this just seemed a bit too silly!

Reply:
I've put this back, it's an intentional statement about the pointlessness of keywords